Source/Core/loadImageFromTypedArray.js

```diff
@@ -0,0 +1,61 @@
+/*global define*/
+define([
+        '../ThirdParty/when',
+        './defined',
+        './DeveloperError'
+    ], function(
+        when,
+        defined,
+        DeveloperError) {
+    "use strict";
+
+    /**
+     * @private
+     */
+    var loadImageFromTypedArray = function(buffer, byteOffset, length, format) {
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(buffer)) {
+            throw new DeveloperError('buffer is required.');
+        }
+
+        if (!defined(byteOffset)) {
+            throw new DeveloperError('byteOffset is required.');
+        }
+
+        if (!defined(length)) {
+            throw new DeveloperError('length is required.');
+        }
+
+        if (!defined(format)) {
+            throw new DeveloperError('format is required.');
+        }
+        //>>includeEnd('debug');
+
+        var bytes = new Uint8Array(buffer, byteOffset, length);
+        var blob = new Blob([bytes], {
+            type : format
+        });
+
+        return when(blob, function(blob) {
```

Comment:
Are you sure?  I'm under the impression that every `createObjectURL` needs a matching `revokeObjectURL` or the memory will never be freed.  MDN seems [to agree](https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL).